pkg/ddc/alluxio/transform_optimization.go

```diff
@@ -50,7 +50,7 @@ func (e *AlluxioEngine) optimizeDefaultProperties(runtime *datav1alpha1.AlluxioR
 	setDefaultProperties(runtime, value, "alluxio.user.block.worker.client.pool.min", "512")
 	setDefaultProperties(runtime, value, "alluxio.fuse.debug.enabled", "false")
 	setDefaultProperties(runtime, value, "alluxio.web.ui.enabled", "false")
-	setDefaultProperties(runtime, value, "alluxio.user.file.writetype.default", "MUST_CACHE")
+	setDefaultProperties(runtime, value, "alluxio.user.file.writetype.default", "ASYNC_THROUGH")
```

Comment:
I think we don't need set the properties, just keep it as alluxio default behavior.

Reply:
The code has been updated.